src/components/transformation/create/index.tsx

```diff
@@ -373,16 +369,32 @@ function TransformationCreate() {
                             }}
                         />
                         <LoadingButton
-                            disabled={!!entityNameError || !!submitButtonError}
                             fullWidth
                             variant="contained"
                             loading={urlLoading}
+                            disabled={
+                                !!entityNameError ||
+                                !!submitButtonError ||
+                                urlLoading
+                            }
                             sx={{ marginBottom: 3 }}
-                            loadingPosition="end"
                             onClick={async () => {
                                 const gitpodUrl = await generateUrl();
                                 if (gitpodUrl) {
-                                    window.open(gitpodUrl, '_blank');
+                                    const gitPodWindow = window.open(
+                                        gitpodUrl,
+                                        '_blank'
+                                    );
+
+                                    if (!gitPodWindow || gitPodWindow.closed) {
+                                        displayError(
+                                            intl.formatMessage({
+                                                id: 'newTransform.errors.gitPodWindow',
+                                            })
+                                        );
+                                    }
```

Comment:
Same check we do with OAuth. Isn't perfect but helps in a situation where the Browser is blocking stuff.